tests/test_external_services/test_tprek/test_tprek_unit_importer.py

```diff
@@ -0,0 +1,126 @@
+import datetime
+
+import pytest
+from django.utils.timezone import get_default_timezone
+
+from common.date_utils import local_datetime
+from spaces.importers.tprek_api_client import TprekAPIClient
+from spaces.importers.tprek_unit_importer import TprekUnitImporter
+from spaces.models import Location
+from tests.factories import UnitFactory
+from tests.helpers import patch_method
+from tests.mocks import MockResponse
+from tests.test_external_services.test_tprek.conftest import SINGLE_TPREK_UNIT_JSON
+from utils.sentry import SentryLogger
+
+# Applied to all tests
+pytestmark = [
+    pytest.mark.django_db,
+]
+
+
+DEFAULT_TIMEZONE = get_default_timezone()
+
+
+@patch_method(TprekAPIClient.get_unit, return_value=(None, None))
+def test_TprekUnitImporter__unit_tprek_id_missing():
+    unit = UnitFactory.create(tprek_id=None)
+
+    importer = TprekUnitImporter()
+    with pytest.raises(ValueError, match=f"Unit TPREK ID is None: {unit.pk}"):
```

Comment:
`match` uses regex matching, so for safety, you should wrap any string intended as a literal match with [re.escape()](https://docs.python.org/3/library/re.html#re.escape) to escape all possible regex patterns in it.